client/apps/Planning/PlanningListSubNav.tsx

```diff
@@ -22,14 +22,17 @@ interface IProps {
     currentInterval: 'DAY' | 'WEEK' | 'MONTH';
     sortOrder: SORT_ORDER;
     sortField: SORT_FIELD;
-
+    userList:Array<{label: string, onSelect(): void}>;
+    users: any;
```

Comment:
Avoid using `any` with types, it is more meaningful and provides better type checking if you use specific types.

Here it would be: `users: Array<IUser>`